readthedocs/core/symlink.py

```diff
@@ -201,10 +218,12 @@ def symlink_subprojects(self):
                 # TODO this should use os.symlink, not a call to shell. For now,
                 # this passes command as a list to be explicit about escaping
                 # characters like spaces.
-                status, _, stderr = run(['ln', '-nsf', docs_dir, symlink])
-                if status > 0:
-                    log.error('Could not symlink path: status=%d error=%s',
-                              status, stderr)
+                result = self.environment.run('ln', '-nsf', docs_dir, symlink)
```

Comment:
This comes from https://github.com/rtfd/readthedocs.org/blob/c37b00995c1bbc5ee51d3552ef176546373bb912/readthedocs/doc_builder/environments.py#L113-L177